product_docs/docs/pem/9/considerations/setup_ha_using_efm.mdx

```diff
@@ -7,13 +7,13 @@ redirects:
 ---
 
 !!! Note
-    This document walks you through setting up the Failover manager for PEM Server with new installation and not with existing installation. The provided commands apply to the configuration on RHEL-based systems where HTTPD is used for the web server services.
+    This procedure is for setting up the failover manager for a PEM server with a new installation, not with an existing one. The provided commands apply to the configuration on RHEL-based systems where HTTPD is used for the web server services.
```

Comment:
Since it is the name of a product (see https://www.enterprisedb.com/docs/efm/latest/), let's capitalize it: 
```suggestion
    This procedure is for setting up the Failover Manager for a PEM server with a new installation, not with an existing one. The provided commands apply to the configuration on RHEL-based systems where HTTPD is used for the web server services.
```

Reply:
@gvasquezvargas If you want to use the proper noun, we need to remove "the." Is it the failover manager or is it Failover Manager?

Reply:
Good question. I guess it is Failover Manager then. 
https://www.enterprisedb.com/docs/efm/latest/architecture/